manifests/install_aem.pp

```diff
@@ -97,15 +101,12 @@
   $aem_jvm_mem_opts              = '-Xss4m -Xmx8192m',
   $aem_keystore_password         = undef,
   $aem_keystore_path             = undef,
+  $aem_ssl_method                = undef,
   $aem_profile                   = 'aem62_sp1_cfp3',
   $aem_sample_content            = false,
   $cert_base_url                 = undef,
   $aem_jvm_opts                  = [
     '-XX:+PrintGCDetails',
-    '-XX:+PrintGCTimeStamps',
-    '-XX:+PrintGCDateStamps',
-    '-XX:+PrintTenuringDistribution',
-    '-XX:+PrintGCApplicationStoppedTime',
```

Comment:
Those values are only not supported in JDK11 right ? please keep them since they are supported for JDK8 and to keep the backwards compatibility.